alleyinteractive/wp-proxy-service
# Issue-5: Proxy response should return only the body or the remote request and set the headers, status, etc.

### Summary
This PR addresses the issue described in https://github.com/alleyinteractive/wp-proxy-service/issues/5. Specifically, it ensures that the proxy response correctly returns only the body of the remote request and appropriately sets the headers, status, etc., as described in the issue. Fixes #5

### Changes
- Update the proxy response handling logic to correctly separate and set the body, headers, status, etc., in the client's response. This includes adjustments to ensure that each piece of the response (body, headers, status) is correctly identified and applied from the remote response to the proxied response.

### How to test
- Make a proxied request as per the steps described in the issue.
- Verify that the response received is only the body of the remote request.
- Ensure that headers and status are correctly set as per the remote response.

### Additional Information
- This PR does not include any changes to existing configurations or dependencies.
